onboard/gps/src/__main__.py

```diff
@@ -121,12 +127,14 @@ async def recieve(self, lcm):
                 continue
 
             lcm.publish('/gps', gps_struct.encode())
-            seen_tags = {tag: False for tag in self.NMEA_TAGS_MAPPER.keys()}
+            seen_tags = {tag: False if not tag == 'TXT' else True
+                         for tag in self.NMEA_TAGS_MAPPER.keys()}
```

Comment:
Actually the opposite, the only tag that I am not looking for is TXT tags, but I have a handler for them in case they show up and the way I have things setup is such that every supported tag is expected to show up regularly so I made a quick hack to exclude TXT. And for clarity TXT statements hold error msgs and that sort of thing

Reply:
This makes sense